Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +91,19 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null || index < 0 || index >= array.Length)
```

Comment:
Many conditions, consider using a bool variable to explain what these conditions mean altogether

Reply:
I moved the chain of conditions to a new bool variable ArgumentsAreNotValid